db/migrate/20230921021333_add_user_to_bookmarks.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToBookmarks < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :bookmarks, :user, null: false, foreign_key: true
```

Comment:
same here bookmarks already has the user_id column, this result in `PG::DuplicateColumn: ERROR: `